health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignDetails.js

```
@@ -113,31 +117,79 @@ const CampaignDetails = () => {
+    "dd/MM/yyyy"
+  )}`;
+
+  const closeToast = () => {
+    setShowToast(null);
+  };
 
   return (
     <>
       <div className="campaign-details-header">
         <HeaderComponent className={"date-header"}>{campaignData?.campaignName}</HeaderComponent>
         <Tag label={campaignData?.campaignName} showIcon={false} className={"campaign-view-tag"} type={"warning"} stroke={true}></Tag>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)